pkg/register/documentBuilder.go

```diff
@@ -82,19 +82,14 @@ func (b RegisterDocumentBuilder) build() (*RegisterDocument, error) {
 		specVersion = b.SpecVersion
 	}
 
-	updateTime := defaultUpdateTimeSeconds()
-	if b.UpdateTime != 0 {
-		updateTime = b.UpdateTime
-	}
-
 	doc := &RegisterDocument{
 		Context:                DocumentContext,
 		ID:                     b.ID,
 		IoticsSpecVersion:      specVersion,
 		IoticsDIDType:          b.Purpose,
 		Controller:             b.Controller,
 		Creator:                b.Creator,
-		UpdateTime:             updateTime,
+		UpdateTime:             defaultUpdateTimeMillis(),
```

Comment:
Is there any impact on existing docs if update time unit changes?
.. or is it a non-issue because e.g. previously time would be `N` and now it would become at least `N * 1000`?

Reply:
It works for existing documents because N * 1000.  I'll add a comment to say milliseconds